python/ray/dashboard/modules/event/tests/test_export_task.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import sys
+
+import pytest
+
+import ray
+
+
+@pytest.mark.asyncio
+async def test_task_labels(tmp_path):
+    """
+    Test task events are correctly generated and written to file
+    """
+    os.environ["RAY_enable_export_api_write"] = "1"
+    ray.init(_temp_dir=str(tmp_path))
```

Comment:
we don't need to set a temp directory, we can retrieve the temporary directory set by ray and find the export_event_path.

See example like
```
def test_event_basic(disable_aiohttp_cache, ray_start_with_dashboard):
    assert wait_until_server_available(ray_start_with_dashboard["webui_url"])
    webui_url = format_web_url(ray_start_with_dashboard["webui_url"])
    session_dir = ray_start_with_dashboard["session_dir"]
    event_dir = os.path.join(session_dir, "logs", "events")
```